### PR TITLE
Increase test coverage in component_build_controller_simple_build.go and

### PR DESCRIPTION
**Pull request by rcerven**
perror.go

### CODE REVIEW

**Review by mmorhun — APPROVED** [review]
LGTM. The only thing is it would be great to have better PR description.

**Comment by mkosiarc** [comment]
There is also typo coverate/coverage in the commit message

File: controllers/component_build_controller_unit_test.go [hunk 1]
```diff
@@ -185,6 +188,128 @@ func TestWriteBuildStatus(t *testing.T) {
...
+
+	_, err := generatePipelineRunForComponent(component, pipelineRef, additionalParams, pRunGitInfo)
+	if err == nil {
+		t.Error("generateInitialPipelineRunForComponentDevfileError(): Didn't return error")
+	} else {
+		gotError := strings.Contains(err.Error(), "failed to populateAndParseDevfile: failed to decode devfile json")
+		if gotError != true {
+			t.Errorf("generateInitialPipelineRunForComponentDevfileError(): Got incorrect error: '%v'", err)
+		}
+	}
```
Comment (tkdchen) [inline controllers/component_build_controller_unit_test.go:240]: How about use `assert.ErrorContains` from package gotest.tools to simplify this if-else?

**Comment by rcerven** [comment]
> There is also typo coverate/coverage in the commit message

fixed

File: controllers/component_build_controller_unit_test.go (hunk 1 above)
Comment (rcerven) [reply controllers/component_build_controller_unit_test.go:238]: changed

**Review by mkosiarc — APPROVED** [review]

**Review by mkosiarc — APPROVED** [review]

**Pull request merged**